packages/containers/src/ComponentForm/ComponentForm.component.js

```diff
@@ -175,14 +175,15 @@ export class TCompForm extends React.Component {
 
 	setupTrigger(props) {
 		const config = cmf.sagas.http.getDefaultConfig() || {};
+		const defaultSecurity = config.security || {};
 		this.trigger = kit.createTriggers({
 			url: props.triggerURL,
 			customRegistry: props.customTriggers,
 			headers: config.headers,
 			lang: props.lang,
 			security: {
-				CSRFTokenCookieKey: props.CSRFTokenCookieKey,
-				CSRFTokenHeaderKey: props.CSRFTokenHeaderKey,
+				CSRFTokenCookieKey: props.CSRFTokenCookieKey || defaultSecurity.CSRFTokenCookieKey,
+				CSRFTokenHeaderKey: props.CSRFTokenHeaderKey || defaultSecurity.CSRFTokenHeaderKey,
```

Comment:
I would prefer the default to be handled in the createTriggers function.

Reply:
Yes, good point